lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -215,7 +215,13 @@ def _set_model_type(self, request: BaseModel) -> str:
 
         return model_url
 
-    def _get_job_params(self, job_type: str, record, request: BaseModel) -> dict:
+    def _validate_config(self, job_type: JobType, config_template: str, config_params: dict):
+        if job_type == JobType.INFERENCE:
+            InferenceJobConfig.model_validate_json(config_template.format(**config_params))
```

Comment:
Note that having some sort of class representing a `JobPackage` would remove this kind of "if job_type == X" checks by just using the appropriate overloaded method, like `job.config_model.model_validate_json(...)`.

Reply:
to be addressed in https://github.com/mozilla-ai/lumigator/issues/706